main.go

```diff
@@ -126,73 +109,251 @@ func (e *periodicExporter) Collect(ch chan<- prometheus.Metric) {
+	e.counter(metricsChan, labels, labelValues, "mesos_master_slave_shutdowns_scheduled_total", float64(stats.SlaveShutdownsScheduled), "Total of slave shutdowns scheduled")
+	e.gauge(metricsChan, labels, labelValues, "mesos_master_slave_active", float64(stats.SlavesActive), "Amount of active slaves")
+	e.counter(metricsChan, labels, labelValues, "mesos_master_slave_connected_total", float64(stats.SlavesConnected), "Total of connected slaved")
+	e.counter(metricsChan, labels, labelValues, "mesos_master_slave_disconnected_total", float64(stats.SlavesDisconnected), "Total of disconnected slaves")
+	e.gauge(metricsChan, labels, labelValues, "mesos_master_slave_inactive", float64(stats.SlavesInactive), "Amount of inactive slaves")
+
+	e.counter(metricsChan, labels, labelValues, "mesos_master_task_failed_source_slave_reason_command_executor_failed_total", float64(stats.TaskFailedSourceSlaveReasonCommandExecutorFailed), "Total tasks failed with as reason the command executor failed")
+
+	e.counter(metricsChan, labels, labelValues, "mesos_master_tasks_error_total", float64(stats.TasksError), "Total tasks in error")
+	e.counter(metricsChan, labels, labelValues, "mesos_master_tasks_failed_total", float64(stats.TasksFailed), "Total of failed tasks")
+	e.counter(metricsChan, labels, labelValues, "mesos_master_tasks_finished_total", float64(stats.TasksFinished), "Total of finished tasks")
+	e.counter(metricsChan, labels, labelValues, "mesos_master_tasks_killed_total", float64(stats.TasksKilled), "Total of killed tasks")
+	e.counter(metricsChan, labels, labelValues, "mesos_master_tasks_lost_total", float64(stats.TasksLost), "Total of lost tasks")
+	e.gauge(metricsChan, labels, labelValues, "mesos_master_tasks_running", float64(stats.TasksRunning), "Amount of running tasks")
+	e.gauge(metricsChan, labels, labelValues, "mesos_master_tasks_staging", float64(stats.TasksStaging), "Amount of staging tasks")
+	e.gauge(metricsChan, labels, labelValues, "mesos_master_tasks_starting", float64(stats.TasksStarting), "Amount of starting tasks")
+
+	e.counter(metricsChan, labels, labelValues, "mesos_master_uptime_seconds", stats.UptimeSecs, "Master uptime in seconds")
+
+	e.counter(metricsChan, labels, labelValues, "mesos_master_valid_framework_to_executor_messages_total", float64(stats.ValidFrameworkToExecutorMessages), "Total of valid framework-to-executor messages")
+	e.counter(metricsChan, labels, labelValues, "mesos_master_valid_status_update_acknowledgements_total", float64(stats.ValidStatusUpdateAcknowledgements), "Total of valid status update acknowledgements")
+	e.counter(metricsChan, labels, labelValues, "mesos_master_valid_status_updates_total", float64(stats.ValidStatusUpdates), "Total of valid status updates")
+
+	e.counter(metricsChan, labels, labelValues, "mesos_master_registrar_queued_operations_total", float64(stats.RegistrarQueuedOperations), "Total of queued operations in registrar")
```

Comment:
The description reads as a though this was the number of operations currently queued